source.php

```diff
@@ -17,1361 +31,1122 @@
-    }
-    //G+ Share Button End
-    //Sort by indexes
-    ksort( $active_services );
-
-    //Form the ordered buttons markup
-    $active_services = implode( '', $active_services );
-    //Rest of the stuff
-    $layout = '<div class="rtsocial-container rtsocial-container-align-' . $options[ 'alignment_options_set' ] . ' rtsocial-' . $options[ 'display_options_set' ] . '">';
-    //Append the ordered buttons
-    $layout .= $active_services;
-    //Hidden permalink
-    $layout .= '<a rel="nofollow" class="perma-link" href="' . get_permalink( $post->ID ) . '" title="' . esc_attr( get_the_title( $post->ID ) ) . '"></a><input type="hidden" name="rts_id" class="rts_id" value="' . $post->ID . '" />' . wp_nonce_field( 'rts_media_' . $post->ID, 'rts_media_nonce', true, false ) . '</div>';
-
-    if ( $options[ 'placement_options_set' ] == 'top' ) {
-        return $layout . $content;
-    } else {
-        if ( $options[ 'placement_options_set' ] == 'bottom' ) {
-            return $content . $layout;
-        } else {
-            return $content;
-        }
-    }
+					}
+				}
+			}
+		}
+
+		$tw_layout .= '</div>';
+
+		$active_services[ $tw ] = $tw_layout;
+	}
+	// Twitter End.
+	// Facebook.
+	if ( ! empty( $options )
+	&& ! empty( $options['active'] )
+	&& in_array( 'fb', $options['active'], true ) ) {
+		$fb          = array_search( 'fb', $options['active'], true );
+		$fb_count    = ( empty( $options['hide_count'] ) || 1 !== (int) $options['hide_count'] ) ? '<div class="rtsocial-' . $options['display_options_set'] . '-count"><div class="rtsocial-' . $options['display_options_set'] . '-notch"></div><span class="rtsocial-fb-count"></span></div>' : '';
+		$class       = '';
+		$rt_fb_style = '';
+		$path        = plugins_url( 'images/', __FILE__ );
+
+		if ( 'like_dark' === $options['fb_style'] ) {
+			$class       = 'rtsocial-fb-like-dark';
+			$rt_fb_style = 'fb-dark';
+		} else {
+
+			if ( 'recommend_dark' === $options['fb_style'] ) {
+				$class       = 'rtsocial-fb-recommend-dark';
+				$rt_fb_style = 'fb-dark';
+			} else {
+
+				if ( 'recommend_light' === $options['fb_style'] ) {
+					$class       = 'rtsocial-fb-recommend-light';
+					$rt_fb_style = 'fb-light';
+				} else {
+
+					if ( 'share' === $options['fb_style'] ) {
+						$class = 'rtsocial-fb-share';
+					} else {
+						$class       = 'rtsocial-fb-like-light';
+						$rt_fb_style = 'fb-light';
+					}
+				}
+			}
+		}
+
+		$fb_layout = '<div class="rtsocial-fb-' . $options['display_options_set'] . ' ' . $rt_fb_style . '">';
+
+		$rt_social_text = '';
+
+		if ( 'like_light' === $options['fb_style']
+		|| 'like_dark' === $options['fb_style'] ) {
+			$rt_social_text = 'Like';
+		} else {
+
+			if ( 'recommend_light' === $options['fb_style']
+			|| 'recommend_dark' === $options['fb_style'] ) {
+				$rt_social_text = 'Recommend';
+			} else {
+				$rt_social_text = 'Share';
+			}
+		}
+
+		if ( 'horizontal' === $options['display_options
```

Comment:
There is no use of this wrapper. Use `filemtime` directly

Reply:
used `filemtime` directly.